client/main.lua

```diff
@@ -70,168 +70,155 @@ local function registerDutyTarget()
-                event = "qb-mechanicjob:client:target:OpenStash",
-                label = Lang:t('labels.o_stash'),
-                icon = 'fa fa-archive'
-            }},
-            distance = 2.0
+        exports.ox_target:addBoxZone({
+            name = "stash",
+            coords = coords,
+            size = size,
+            rotation = 340.75,
+            debug = Config.DebugPoly,
+            drawSprite = true,
+            options = {
+                {
+                    event = "qb-mechanicjob:client:target:OpenStash",
+                    label = Lang:t('labels.o_stash'),
+                    icon = 'fa fa-archive',
+                    groups = 'mechanic',
+                    distance = 2.0
+                }
+            }
         })
 
         Config.Targets[stashTargetBoxId] = {created = true}
     else
-        local zone = BoxZone:Create(coords, 1.5, 1.5, {
-            name = stashTargetBoxId,
-            heading = 0,
-            debugPoly = Config.DebugPoly,
-            minZ = coords.z - 1.0,
-            maxZ = coords.z + 1.0,
-        })
-        zone:onPlayerInOut(function (isPointInside)
-            if isPointInside then
-                lib.showTextUI(Lang:t('labels.o_stash'), {
-                    position = 'left'
-                })
-            else
-                lib.hideTextU()
+        lib.zones.box({
+            coords = coords,
+            size = size,
+            rotation = 340.75,
+            debug = Config.DebugPoly,
+            onEnter = function()
+                isInsideStashZone = true
+                lib.showTextUI('[E] ' .. Lang:t('labels.o_stash'), { position = 'left-center' })
+            end,
+            onExit = function()
+                lib.hideTextUI()
+                isInsideStashZone = false
             end
-
-            isInsideStashZone = isPointInside
-        end)
-
+        })
         Config.Targets[stashTargetBoxId] = {created = true, zone = zone}
     end
 end
 
 local function registerGarageZone()
     local coords = Config.Locations.vehicle
-    local vehicleZone = BoxZone:Create(coords.xyz, 5, 15, {
-        name = 'vehicleZone',
-        heading = 340.0,
-        minZ = coords.z - 1.0,
-        maxZ = coords.z + 5.0,
-        debugPoly = false
-    })
-
-    vehicleZone:onPlayerInOut(function (isPointInside)
-        if isPointInside and QBX.PlayerData.job.onduty then
-            local inVehicle = cache.vehicle
-            if inVehicle then
-                lib.showTextUI(Lang:t('labels.h_vehicle'), {
-                    position = 'left'
-                })
-            else
-                lib.showTextUI(Lang:t('labels.g_vehicle'), {
-                    position = 'left'
-                })
+    local size = Config.LocationsSize.vehicle
+
+    lib.zones.box({
+        coords = coords,
+        size = size,
+        rotation = 339.75,
+        debug = Config.DebugPoly,
+        onEnter = function()
+            if QBX.PlayerData.job.onduty then
```

Comment:
Good opportunity for a guard clause; https://manason.github.io/effective-fivem-lua/functions/#avoid-nesting

Reply:
I did changes with the guide u provided can u check it